templates/bootstrap.sh

```diff
@@ -463,7 +463,7 @@ else
     fi
 
     # Run the build script command in the service specified in BUILDKITE_DOCKER_COMPOSE_CONTAINER
-    echo "+++ $BUILDKITE_COMMAND_ACTION (in Docker Compose container)"
+    echo "~~~ $BUILDKITE_COMMAND_ACTION (in Docker Compose container)"
```

Comment:
Ah, I must not have been testing it properly back in the day… from memory it wasn't automatically expanding for simple commands, such as running `ls` or things if people hadn't got the stage of adding `echo '---'` style output headers. Not sure why it wasn't working back then! I'll leave this to you 👍